docs/products/postgresql/concepts/pg-audit-logging.rst

```diff
@@ -0,0 +1,132 @@
+Aiven for PostgreSQL audit logging
+==================================
+
+.. important::
+
+   Aiven for PostgreSQL® audit logging is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+
+Discover the Aiven for PostgreSQL® audit logging feature and its capabilities. Learn why use it and how it works.
+
+.. seealso::
+
+   For information on how to enable the audit logging on your Aiven for PostgreSQL service and how to access and visualize your logs, check out :doc:`Collect audit logs in Aiven for PostgreSQL® </docs/products/postgresql/howto/pgaudit-logging>`.
+
+About audit logging
+-------------------
+
+The audit logging feature allows you to monitor and track activities within rellational database systems, which helps you cope with unauthorized access, data breaches, or fraudulent activities ensuring data integrity and security. Using audit logs enables you to maintain compliance with regulations and standards set by a government or an industry. The audit logging is also a tool for investigating and resolving data discrepancies, troubleshooting system errors, and optimizing database performance. For more applications of the audit logging feature, check out :ref:`Why use the audit logging <why-use-pgaudit>`.
+
+.. _why-use-pgaudit:
+
+Why use audit logging
+---------------------
+
+There are multiple reasons why you may want to use the audit logging. Note the following as benefits of using this feature:
+
+Data Security
+  * Monitor user activities to identify unusual or suspicious behavior
+  * Detect unauthorized access attempts to critical data or systems
+  * Identify intrusion attempts or unauthorized activities within the organization's IT environment
```

Comment:
To help with the conversion, could I ask you to fix the indent for each category on this page? Feel free to format as you want, but we don't want indentation 🙂  

You can also do this in one operation by searching for `  *` and replacing by `*` in vscode.

```suggestion
**Data Security:**
* Monitor user activities to identify unusual or suspicious behavior
* Detect unauthorized access attempts to critical data or systems
* Identify intrusion attempts or unauthorized activities within the organization's IT environment
```